apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLoggerMiddleware.cs

```diff
@@ -28,16 +28,14 @@ namespace Google.Cloud.Diagnostics.AspNetCore
     public sealed class ErrorReportingExceptionLoggerMiddleware
     {
         private readonly RequestDelegate _next;
-        private readonly IExceptionLogger<ReportErrorEventResponse> _logger;
+        private readonly IExceptionLogger _logger;
 
         /// <summary>
         /// Create a new instance of <see cref="ErrorReportingExceptionLoggerMiddleware"/>.
         /// </summary>
         /// <param name="next">The next request delegate. Cannot be null.</param>
-        /// <param name="logger">A logger that will report exceptions to the Stackdriver
-        ///     Error Reporting API. Cannot be null.</param>
-        public ErrorReportingExceptionLoggerMiddleware(
-            RequestDelegate next, IExceptionLogger<ReportErrorEventResponse> logger)
+        /// <param name="logger">A logger that will report exceptions. Cannot be null.</param>
+        internal ErrorReportingExceptionLoggerMiddleware(RequestDelegate next, IExceptionLogger logger)
```

Comment:
Opp yes we do otherwise we'll fail to instantiate it with dependency injection.  Fixed.